test/metabase/search/postgres/index_test.clj

```diff
@@ -32,17 +34,18 @@
   "Ensure a clean, small index."
   [& body]
   `(when (= :postgres (mdb/db-type))
-     (mt/dataset ~(symbol "test-data")
-       (mt/with-temp [:model/Card     {}           {:name "Customer Satisfaction" :collection_id 1}
-                      :model/Card     {}           {:name "The Latest Revenue Projections" :collection_id 1}
-                      :model/Card     {}           {:name "Projected Revenue" :collection_id 1}
-                      :model/Card     {}           {:name "Employee Satisfaction" :collection_id 1}
-                      :model/Card     {}           {:name "Projected Satisfaction" :collection_id 1}
-                      :model/Database {db-id# :id} {:name "Indexed Database"}
-                      :model/Table    {}           {:name "Indexed Table", :db_id db-id#}]
-         (search.index/reset-index!)
-         (search.ingestion/populate-index!)
-         ~@body))))
+     (binding [search.ingest]
```

Comment:
🤔 what does this mean though?

Reply:
This means I didn't stage my code properly 😂